CHANGES.md

```diff
@@ -1,3 +1,18 @@
+Release 4.41 (release date: 2019-07-02)
+=======================================
+New Features
+------------------
+- Added HMAC key management for service accounts. This includes a new "hmac"
+  command to create/get/update/delete HMAC on service accounts. Currently,
+  your project must be whitelisted for use with the new BPO functionality.
```

Comment:
Replace "with the new BPO functionality" with "with this new HMAC functionality"